### DISCUSSION TITLE
Please use the automatic login from your Nextcloud

### DISCUSSION THREAD

**Discussion by mrozj** [nextcloud/all-in-one#1072] [Questions] [answered]
Just used the Docker compose file to create a new installation of Nextcloud. I am also running NC on an Unraid server but don't care for the performance. I stood up an Ubuntu VM installation and used the compose file to install Nextcloud AIO. The installation works flawlessly. Logging in is a different issue. I get this error:
The login is blocked since Nextcloud is running. Please use the automatic login from your Nextcloud.

![image](https://user-images.githubusercontent.com/92899646/187013976-5a7ddaba-5fab-4501-81a9-00e02eb37e10.png)

You can unblock the login by running 'sudo docker stop nextcloud-aio-apache'.

I am running Nginx Proxy Manager but I get the same error when I access the site using the local IP address. 

Be kind in your responses. I am using this as a learning experience. My background is as a Windows Server admin.

**Comment by Zoey2936** [comment]
Just open your cloud (not the aio pannel) and then go to:
Your Profile Picture in the upper left, click on it, than settings, than on the left search for overview, there you have this button
![grafik](https://user-images.githubusercontent.com/75573284/187019577-21358f15-f883-4204-a91c-fedaa17ceb11.png)

**Reply by mrozj** [reply]
Thanks Zoey but I get that message when I click on 
![image](https://user-images.githubusercontent.com/92899646/187037583-b88e9c82-62fb-40d7-9f8f-08925e3e8170.png)

I don't see Open Nextcloud AIO Interface

**Reply by Zoey2936** [reply]
But the screenshot you sent is the aio interface?

**Reply by szaimen** [reply]
Sounds like https://help.nextcloud.com/t/nextcloud-aio-install-location-of-the-backup-archive/142893/5

**Reply by mrozj** [reply]
Sorry. Maybe I am not being clear. When I click on this:
![image](https://user-images.githubusercontent.com/92899646/187038053-747fb596-2608-460b-88b6-b0bf2efb41b0.png)
I get this:
![image](https://user-images.githubusercontent.com/92899646/187038070-e73f9acb-8369-40d3-be5e-8649ae1373e2.png)

Not sure where I go from here.

**Reply by szaimen** [reply]
Based on the link that I posted, your firewall may be disabled which leads to this behaviour. You need to enable your firewall in order to fix this.

**Reply by mrozj** [reply]
Thanks for your patience szaimen. I enabled the firewall just now with these commands:
sudo ufw enable
sudo ufw allow 80/tcp
sudo ufw allow 443/tcp
sudo ufw allow 8080/tcp
sudo ufw allow 8443/tcp
sudo ufw allow 3478

also 22 and 11000 (not sure if I needed 11000 for apache)

Regardless they didn't help even after a reboot and now I cannot even reach the login page nor can I SSH to the server. 

I guess this is why they make Snap installers. LOL

**Reply by szaimen** [reply]
usually it works without any problem. Seems like your are simply unlucky.

**Reply by mrozj** [reply]
Ok, I missed adding 22 for SSH. I added it and I can SSH to the server. 9000 is entered as well and I cannot access Portainer.

**Comment by szaimen** [comment, marked as answer]
See also https://github.com/nextcloud/all-in-one#how-to-easily-log-in-to-the-aio-interface

**Reply by mrozj** [reply]
Thanks but I changed my URL to add: settings/admin/overview but I end up at the same page.

**Reply by szaimen** [reply]
Sounds like https://help.nextcloud.com/t/nextcloud-aio-install-location-of-the-backup-archive/142893/5

**Reply by szaimen** [reply]
Based on the link that I posted, your firewall may be disabled which leads to this behaviour. You need to enable your firewall in order to fix this.

**Comment by Stooovie** [comment]
Started happening after upgrading to AIO 9.X. Impossible to get into the AIO screen with either Wan or direct LAN address. No changes on my part other than the upgrade. 

Recommended way to use https://yourdomain.tld/settings/admin/overview
leads to the same screen.

**Reply by szaimen** [reply]
This was solved in https://help.nextcloud.com/t/aio-login-with-9-0-1-the-login-is-blocked-since-nextcloud-is-running/195507

**Comment by Lowitle** [comment]
I'm having the same issue with a new installation.

Nextcloud AIO v9.6.0

Tried to restart.
Tried to disable all the port rules on my firewall.
Tried to a clean re-install.

Always the same issue.

I have the machine installed in Docker, Synology NAS.

Installed via SSH using this command:
```
sudo docker run \
--init \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 8080:8080 \
--env APACHE_PORT=11000 \
--env APACHE_IP_BINDING=[0.0.0.0](https://0.0.0.0/) \
--env SKIP_DOMAIN_VALIDATION=true \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /volume1/docker/docker.sock:/var/run/docker.sock:ro \
--env WATCHTOWER_DOCKER_SOCKET_PATH="/volume1/docker/docker.sock" \
nextcloud/all-in-one:latest
```

I'm not gonna give up. I'm googling and the 2 solutions I found, nothing worked.

¿Any ideas?

**Reply by szaimen** [reply]
Did you point your reverse proxy at port 11000 like shown in the documentation?

**Reply by Lowitle** [reply]
Thx! I wrong used the 8080. That was the reason to all my problems!
